core/src/display_object.rs

```diff
@@ -951,6 +1034,14 @@ pub trait TDisplayObject<'gc>:
         self.base_mut(gc_context).set_maskee(node);
     }
 
+    fn scroll_rect(&self) -> Option<Rectangle> {
+        self.base().scroll_rect
+    }
+
+    fn set_scroll_rect(&self, gc_context: MutationContext<'gc, '_>, rect: Option<Rectangle>) {
+        self.base_mut(gc_context).scroll_rect = rect;
```

Comment:
`scrollRect` is always in units of pixels:

```
var rect = new flash.geom.Rectangle(10.1, 10.1, 1.4, 1.4);
trace(rect); // (x=10.1, y=10.1, w=1.4, h=1.4)
clip.scrollRect = rect;
trace(clip.scrollRect); // (x=10, y=10, w=2, h=2)
```

so I think this setter would be a good place to round the values, so that any later get has the value correctly rounded.